prusti-tests/tests/cargo_verify/veribetrfs/Cargo.toml

```
@@ -0,0 +1,8 @@
+[package]
+name = "veribetrfs"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
```

```suggestion
[dependencies]
prusti-contracts = { path = "prusti-contracts" }
```

I expect the CI to fail if this is missing.

---

I added the dependency now.

---

After adding the dependency, running Prusti locally fails, as there is no dependency "prusti-contracts" in that path.
I tried adding symbolic links but it got messy really quickly, as the transitive dependencies of all the other "prusti-*" crates also are searched in that directory.

Is there an easy way to fix this?

---

When running the tests (`./x.py test` to run all tests, `./x.py test -p prusti-tests --test cargotest` to run just the `cargo_verify` tests) it's the test suite that creates those symbolic links. These commands are what CI (the continuous-integration tests run by GitHub) executes.

If you want to launch `cargo build/prusti` on your own, it should be enough to create the symbolic links to these five folders: https://github.com/viperproject/prusti-dev/blob/dacab2d43007ed6eeb88d87d2a04b061f8684f91/prusti-tests/tests/cargotest.rs#L141-L145 Does that work?

---

yes, that works